bbc/simorgh
# Remove `InlineLink` client-side routing logic

Overall changes
======
Removes the client-side routing logic for links to Optimo Article pages from the `InlineLink` components, as we no longer want to support CSR.

Code changes
======
- Removes CSR logic from both legacy and new `InlineLink` components, in favour of standard anchor tags
- Removes tests targeting CSR

Testing
======
1. Visit http://localhost:7080/hausa/labarai-65766176?renderer_env=live
2. Click the link "Ka'idodjin shiga gasar Hikayata 2023" in the body text
3. Confirm that a full page reload is performed 
**NOTE:** the page will 404, but that is because its running locally

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
